onnxscript/function_libs/torch_aten/ops/nn.py

```diff
@@ -239,17 +239,26 @@ def aten_conv_depthwise3d(
     raise NotImplementedError()
 
 
-def aten_cross_entropy_loss(
-    self: TensorType,
-    target: TensorType,
-    weight: Optional[TensorType] = None,
-    reduction: int = 1,
-    ignore_index: INT64 = -100,
+@torch_op("aten::cross_entropy")
+def aten_cross_entropy(
+    self: TFloatOrBFloat16,
+    target: Sequence[int],
+    weight: Optional[TFloatOrBFloat16] = None,
+    reduction: str = "mean",
+    ignore_index: int = -100,
     label_smoothing: float = 0.0,
-) -> TensorType:
+) -> TFloatOrBFloat16:
     # cross_entropy_loss(Tensor self, Tensor target, Tensor? weight=None, int reduction=Mean, SymInt ignore_index=-100, float label_smoothing=0.0) -> Tensor
 
-    raise NotImplementedError()
+    if op.OptionalHasElement(weight):
+        result, _ = op.SoftmaxCrossEntropyLoss(
+            self, target, reduction=reduction, ignore_index=ignore_index
+        )
+    else:
+        result, _ = op.SoftmaxCrossEntropyLoss(
+            self, target, weight, reduction=reduction, ignore_index=ignore_index
```

Comment:
Do we need optional get element?

Reply:
I think it serves `check None` situation for weight according to what we have in symbolc_opset12.py.

```python
@_onnx_symbolic("aten::cross_entropy_loss")
@_beartype.beartype
def cross_entropy_loss(
    g: jit_utils.GraphContext,
    self,
    target,
    weight,
    reduction,
    ignore_index,
    label_smoothing,
):
    # none reduction : onnx::Constant[value={0}]
    # mean reduction : onnx::Constant[value={1}]
    # sum reduction : onnx::Constant[value={2}]
    reduction = symbolic_helper._maybe_get_const(reduction, "i")
    reduction_vals = ["none", "mean", "sum"]
    reduction = reduction_vals[reduction]

    label_smoothing = symbolic_helper._maybe_get_const(label_smoothing, "f")
    if label_smoothing is not None and label_smoothing > 0.0:
        raise errors.SymbolicValueError(
            "Unsupported: ONNX does not support label_smoothing", self
        )

    # in onnx SoftmaxCrossEntropyLoss specification, ignore_index is optional without default value.
    # therefore we need to set ignore_index attribute even if it is not specified (e.g. ignore_index=-100).
    ignore_index = symbolic_helper._maybe_get_const(ignore_index, "i")
    if weight.node().mustBeNone():
        celoss = g.op(
            "SoftmaxCrossEntropyLoss",
            self,
            target,
            reduction_s=reduction,
            ignore_index_i=ignore_index,
        )
    else:
        celoss = g.op(
            "SoftmaxCrossEntropyLoss",
            self,
            target,
            weight,
            reduction_s=reduction,
            ignore_index_i=ignore_index,
        )

    return celoss
```

Reply:
OptionalGetElement() will throw exception on local box testing